pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/ControllerPeriodicTasksIntegrationTests.java

```diff
@@ -386,7 +395,75 @@ public void afterRealtimeSegmentRelocatorTest(ITestContext context) throws Excep
     dropRealtimeTable(relocationTable);
   }
 
-  // TODO: tests for other ControllerPeriodicTasks (RetentionManager, BrokerValidationManager, OfflineSegmentIntervalChecker, RealtimeSegmentValidationManager)
+  @BeforeGroups(groups = "brokerResourceValidationManager")
```

Comment:
An @AfterGroups method to drop the tables would be great (both table1 and table2 and also any instances that might have been created). There might be tests added which run after this one, so better to restore the state to initial state

Reply:
Done.